src/dailyLog/dailyLog.js

```diff
@@ -51,66 +67,226 @@ function submitButtonClick(record) {
         record.field1 = updateField1.value;
     }
     const updateField2 = document.querySelector("#reflection");
-    // check to see if field1 is null and update record if not
+    // check to see if field2 is null and update record if not
     if (updateField2.value) {
         record.field2 = updateField2.value;
     }
     const updateHours = document.querySelector("#hours");
-    // check to see if field1 is null and update record if not
+    // check to see if hours is null and update record if not
     if (updateHours.value) {
         record.hours = updateHours.value;
     }
-    // update record in the lcoal storage
+    // get the paragraph tag with accomplishments listes in it
+    const hasAccomplishments = document.querySelector(
+        ".js-accomplishment-list"
+    );
+
+    // if accomplishments paragraph is not empty, update the hasAccomplishment variable in the record
+    // and store accomplishments in the local storage
+    if (hasAccomplishments.textContent !== "") {
+        record.hasAccomplishment = true;
+        // content array to hold all the accomplishments
+        const content = [];
+        // gets all the list items from the accomplishment paragraph
+        const listItems = hasAccomplishments.querySelectorAll("li");
+        // loop over the list items and append the textconetnt of the firstChild into the content array
+        for (let item of listItems) {
```

Comment:
change `let` to be `const` in line 93  and there's a typo in line 92 "textcnetnt"